lib/search-bar/index.tsx

```diff
@@ -59,15 +68,21 @@ const mapStateToProps: S.MapState<StateProps> = ({
   showTrash,
 });
 
-const mapDispatchToProps = (dispatch, { noteBucket, onNoteOpened }) => ({
+const mapDispatchToProps: S.MapDispatch<DispatchProps, OwnProps> = (
+  dispatch,
+  { noteBucket, onNoteOpened }
```

Comment:
perfect use of `OwnProps` in the type signature since we're relying on these two passed-in props